compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -34,10 +34,10 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(BATCH_MATMUL, CircleBatchMatMulGraphBuilder);                                // 126
   CIRCLE_NODE(BATCH_TO_SPACE_ND, CircleBatchToSpaceNDGraphBuilder);                        // 37
   CIRCLE_NODE(CAST, CircleCastGraphBuilder);                                               // 53
-  CIRCLE_NODE(CUSTOM, CircleCustomGraphBuilder);                                           // 32
```

Comment:
These seems to be from old ones.

Reply:
Somehow these lines are still in incorrect order in latest master: https://github.com/Samsung/ONE/blob/27381fb7647c097b36c073d80cccc691d0a2f9ee/compiler/luci/import/src/GraphBuilderRegistry.cpp#L37

Should I remove these changes from this PR?

Reply:
Ah...

> Should I remove these changes from this PR?

I think it would be better to do in another PR for sorting :)